recipes-core/lcd-tools/lcd-tools/lcd-tools.cpp

```diff
@@ -0,0 +1,23 @@
+#include "lcd-tools.h"
+
+#include <cxxopts.hpp>
+
+using namespace AsteroidOS::LCD_Tools;
+
+int main(int argc, char** argv){
+	cxxopts::Options options("lcd-tools", "Various controls for the TicWatch LCD");
```

Comment:
I'm a bit concerned about the name of the tool. "LCD tools" is a very generic name. Do these overlay have any special name ? Like HUD or something ? 

If we can't come up with a better name, I'll create a lcd-tools repository under the AsteroidOS org under which this could be hosted.

Reply:
Well it is intended to be a general LCD interface tool that AOS can use to interface. Is a general repository under AOS ok? My concern is that the majority of watches will not have this interface, and if we add a different watch, it will be using different sources. I guess the latter we can solve with specifying a cmake option.

Reply:
Maybe the recipe name should be prefixed with `aos` or `asteroidos`?